src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+(function() {
+    let translations = {
+            'Passwords'                           : 'Mots de passe',
+            'All'                                 : 'Tous',
+            'Folders'                             : 'Dossiers',
+            'Recent'                              : 'Récent',
+            'Favorites'                           : 'Favoris',
+            'Shared'                              : 'Partagés',
+            'Tags'                                : 'Mots clés',
+            'Security'                            : 'Sécurité',
+            'Trash'                               : 'Corbeille',
+            'Details'                             : 'Détails',
+            'Edit'                                : 'Modifier',
+            'Rename'                              : 'Renommer',
+            'Delete'                              : 'Effacer',
+            'Restore'                             : 'Restaurer',
+            'New Password'                        : 'Nouveau mot de passe',
+            'New Folder'                          : 'Nouveau dossier',
+            'New Tag'                             : 'Nouveau mot clé',
+            'Secure'                              : 'Sécurisé',
+            'Weak'                                : 'Faible',
+            'Weak (Duplicate)'                    : 'Faible (Dupliqué)',
+            'Weak (Outdated)'                     : 'Faible (Obsolète)',
+            'Breached'                            : 'Compromis',
+            'Notes'                               : 'Notes',
+            'Share'                               : 'Partager',
+            'Revisions'                           : 'Révisions',
+            'QR Code'                             : 'QR Code',
+            'Password'                            : 'Mot de passe',
+            'Username'                            : 'Nom d’utilisateur',
+            'Website'                             : 'Site internet',
+            'Add Tags...'                         : 'Ajouter des mots clés…',
+            'Color'                               : 'Couleur',
+            'Name'                                : 'Nom',
+            'Label'                               : 'Étiquette',
```

Comment:
I am a bit confused for this one because I cannot find it appears in the app screens...
If it is unrelated to the tags, then the translation could be `Label` or `Libéllé` (both are valid so pick the one you prefer 😉)